src/server/api/v1/user.ts

```diff
@@ -67,42 +67,70 @@ router.get('/info', async (_req, res) => {
 //Working in progress
 router.get('/transactions/:opid', async (req, res) => {
 	try {
-		const transactions = await Transaction.find({ _id: req.params.opid })
+		const transactions = await Transaction.findOne({ _id: req.params.opid })
+		if (!transactions) throw 'NotFound'
 		/** checa se o usuario da transação é o mesmo que esta logado */
-		if (transactions[0].user !== req.user?.id) throw 'NotAuthorized'
+		if (transactions.user.toHexString() !== req.user?.id.toHexString()) throw 'NotAuthorized'
+
 		const tx = {
-			status:        transactions[0].status,
-			currency:      transactions[0].currency,
-			account:       transactions[0].account,
-			amount:       +transactions[0].amount.toFullString(),
-			type:          transactions[0].type,
-			timestamp:     transactions[0].timestamp.getTime()
+			status:        transactions.status,
+			currency:      transactions.currency,
+			txid:		   transactions.txid,
+			account:       transactions.account,
+			amount:        transactions.amount.toFullString(),
+			type:          transactions.type,
+			confirmations: transactions.confirmations,
+			timestamp:     transactions.timestamp.getTime()
 		}
 		res.send(tx)
 	} catch(err) {
-		res.status(401).send({
-			error: 'NotAuthorized',
-			message: 'This transaction does not belong to your account'
-		})
+		if (err === 'NotFound') {
+			res.status(404).send({
+				error: err,
+				message: 'This transaction does not belong to your account'
+			})
+		} else {
+			res.status(401).send({
+				error: err,
+				message: 'This transaction does not belong to your account'
+			})
+		}
 	}
 })
 
-//TODO: consertar o erro "expected undefined to be an object"
+/**
+ * Pega multiplas transações feitas pelo usuario
+ */
 router.get('/transactions', async (req, res) => {
 	/** Indica o numero de transações que sera puladas */
 	const skip: number = +req.query.skip || 0
```

Comment:
Se skip for não-numérico isso vai retornar um NaN, que vai fazer skip ser NaN e não 0, que ctz q vai dar algum problema em algum lugar

Reply:
Ele vai retornar  0 mesmo se for NaN, eu testei só para ter certeza
![image](https://user-images.githubusercontent.com/28350465/77344284-b5314c80-6d11-11ea-836c-770f33a3d276.png)

Reply:
Ah, então excelente